index.html

```diff
@@ -0,0 +1,119 @@
+<!doctype html>
```

Comment:
NBD: If possibly, you should publish this to the (`gh-pages`](https://help.github.com/articles/creating-project-pages-manually/) branch. This would let people play with it without having to clone anything.

http://rafeca.com/2012/01/17/automate-your-release-flow/ (step 5) has a neat way to keep the histories of the two branches related to each other.